docs/src/main/asciidoc/datasource.adoc

```diff
@@ -40,6 +40,21 @@ please refer to the link:reactive-sql-clients[Reactive SQL clients guide].
 This is a quick introduction to datasource configuration.
 If you want a better understanding of how all this works, this guide has a lot more information in the subsequent paragraphs.
 
+=== Zero Config Setup (DevDB)
+
+When testing or running in dev mode Quarkus can even provide you with a zero config database out of the box, a feature
+we refer to as DevDB. DevDB is supported for the following open source databases:
+
+* Postgresql
+* MySQL
+* MariaDB
+* H2
+* Apache Derby
+
+If you want to use DevDB then all you need to do is include the relevant extension for the type of database you want (either reactive or
```

Comment:
do i read this right that we'll automatically enable testcontainers in devmode ? 

what about users that do not have docker availlable ? or those who set these things outside quarkus ?

Reply:
If you manually configure a database then everything behaves the same way it currently does. This won't affect existing applications, except for the odd corner case of having database extensions installed but not actually using a database. 

If you don't have docker then you need to use h2 if you want to use DevDB, or just configure a database.

Reply:
How does DevDB behave when a `@QuarkusTestResource` is used that has configured a database?

Reply:
If you have configured a db it won't start. This removes the need for a lot of QuarkusTestResource stuff. I could have deleted them but we also need to test explicit configuration.

Reply:
Right, I wanted to make sure that this work doesn't break existing `@QuarkusTestResource`.

I agree that in the future a lot of that stuff will become irrelevant